### ISSUE TITLE
Problems installing local development environment, improve documentation regarding API Umbrella

### ISSUE THREAD

**Issue by jykae** [apinf/api-umbrella-dashboard#535]
I started installing APInf components yesterday. All the API Umbrella packages are for Linux. What is the recommended way to install Umbrella for Mac OSX, Windows?

Tried first to install .deb package with Vagrant on fresh Ubuntu14.04 VM. Had problems with port number in URL address. When going to admin page, Umbrella rewrites URL. Should I forward HTTPS traffic also in Vagrant?

**Comment by jykae** [comment]
I update Vagrantfile & shell provisioning for this as I install forward. I am running on Mac OSX.

**Comment by jykae** [comment]
Discussion in the community https://community.apiumbrella.io/t/vagrant-support/31/2

**Comment by jykae** [comment]
I solved redirect problem, configured private network for VM, access for host-machine only, which is good for local development.

config.vm.network "private_network", ip: "192.168.33.10"

**Comment by jykae** [comment]
Didn't get Github login work, error with callback URL. 
Persona login works. Got logged in admin panel.

**Comment by jykae** [comment]
Maybe it would be good to have Vagrantfile & provisioning scripts in another repo? And in provisioning shell script clone api-umbrella-dashboard from Github. I try to do it that way..

**Comment by jykae** [comment]
Meteor has issues working inside Virtualbox. For development probably then Virtualbox for Umbrella & local installation of APInf Dashboard.

Wondering where I get "auth_token" for Umbrella? Registered API key, how to get "auth_token" for Meteor?

Command "meteor --settings settings.json" fails now with error:

Error: failed [401] {"error":"You need to sign in or sign up before continuing."}

And continues trying to build in a loop.

**Comment by frenchbread** [comment]
@JuusoV Check you email. I've sent you instructions how to get API token for apiUmbrella. All the links are there as well. Notice, that you must login to apiUmbrella's admin panel using the same email which you used to get an API Key.

**Comment by brylie** [comment]
@jykae, please review the API Umbrella [Admin API documentation](http://apiumbrella.io/docs/admin-api/), which describes how to get the admin auth token.

**Comment by frenchbread** [comment]
@brylie Is that right that for team members we do not need to download API Umbrella instance locally and can actually use one that is attached to APInf during testing/development ?

**Comment by brylie** [comment]
@frenchbread yes. We have been using our deployed version of API Umbrella (nightly.apinf.io), since we have been experiencing difficulties with Vagrant, etc.

**Comment by jykae** [comment]
@frenchbread Thanks for instructions and help. I got Umbrella now also working locally with APInf frontend. I am running local Umbrella inside Virtualbox built with Vagrant. Looks like I just had to generate "auth_token" using mongo client after Umbrella 0.8.0 installation from .deb package. Commented here about it: https://github.com/NREL/api-umbrella/issues/135

I document usage of Umbrella Vagrant box in https://github.com/apinf/apinf-devops and auth_token generation after installation.

Meteor however seems to run better natively, as database transactions don't work inside Virtualbox.

This issue can be closed now.
